website/content/docs/operators/deployment/schedulers/mesos-local-mac.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Setting up Heron with Mesos Cluster Locally on Mac
+---
+
+This is a step by step guide to run Heron on a Mesos cluster locally.
+
+## Install Heron
+Follow [Quick Start Guide](../../../../getting-started) to install Heron.
+
+## Setting up an Apache Mesos Cluster Locally
+
+Follow [Installing Mesos on your Mac with Homebrew](https://mesosphere.com/blog/2014/07/07/installing-mesos-on-your-mac-with-homebrew/) to have Mesos installed and running. To confirm Mesos cluster is ready for accepting Heron topologies, make sure you can access Mesos management console [http://localhost:5050](http://localhost:5050) and see activated slaves.
+
+![console page](/img/mesos-management-console.png)
+
+## Configure Heron
+
+### State Manager
+By default, Heron uses Local File System State Manager on Mesos to manage states. You can modify `$HOME/.heron/conf/mesos/statemgr.yaml` to use ZooKeeper. For more details please see [Setting up ZooKeeper](../statemanagers/zookeeper).
+
+### Scheduler
+Heron needs to know where to load the lib to interact with Mesos. Change the config `heron.mesos.native.library.path` in `$HOME/.heron/conf/mesos/scheduler.yaml` to the library path of your Mesos. If Mesos is installed through `brew`, the library path should be `/usr/local/Cellar/mesos/your_mesos_version/lib`.
+
+> Mesos only offers interface in C++ library, which is not portable across platforms.
+
+
+## Run Topology in Mesos
+
+After setting up Heron and Mesos, submit a topology to test if everything works right. By default this command loads the config in `$HOME/.heron/conf`, add `--config-path=your_conf_path` to change the config path.
```

Comment:
s/to test if everything works right/using the following command/

s/, add/. Add/